pytest_logger/plugin.py

```diff
@@ -174,6 +179,14 @@ def add_loggers(self, loggers, stdout_level=logging.NOTSET, file_level=logging.N
         self._enabled = True
         self._loggers.append((loggers, _sanitize_level(stdout_level), _sanitize_level(file_level)))
 
+    def set_formatter(self, formatter):
+        """Sets the `logging.Formatter` to be used by all loggers.
+
+        :arg formatter: The `logging.Formatter` object
+        """
+        if isinstance(formatter, logging.Formatter):
```

Comment:
Just like in other methods of LoggerConfig, I'd assume user passes correct type. Maybe assert would be in place, but I wouldn't go with a silent if (this way user who puts there something incorrect doesn't get a visible error).

Reply:
Gone in 666aa66